op-mode-definitions/monitor-traceroute.xml.in

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="monitor">
+    <properties>
+      <help>Monitor route with mtr</help>
+    </properties>
+    <children>
```

Comment:
This can't be build, it results in:

```
Opening and ending tag mismatch: children line 7 and interfaceDefinition, line 28, column 23 (monitor-traceroute.xml, line 28)
```

Please also use two spaces as indent.